lib/tochtli/reply_queue.rb

```diff
@@ -56,7 +56,7 @@ def register_message_handler(message, handler=nil, timeout=nil, &block)
 
     def on_delivery(delivery_info, metadata, payload)
       class_name       = metadata.type.camelize.gsub(/[^a-zA-Z0-9\:]/, '_') # basic sanity
-      reply_class      = eval(class_name)
+      reply_class      = Object.const_get(class_name)
```

Comment:
Better. Let's put it into separate helper method (with descriptive name ;) ) and it will bo OK for me.